City-of-Helsinki/spouse-program
# Add custom plugin to force email-from settings.

Relates to [ZEN-1129](https://druid.zendesk.com/agent/tickets/1129)

Email from headers are bonkers in Password reset email.
This fix will set Email From headers to no-reply@spouseprogram.fi

Testing Guide: 
Create new user, check from mailhog that it sends email to admin and user (and from headers are sane)
As a new user order new password at: https://spouse.docker.sh/wp-login.php?action=lostpassword
Check that new mail is sent with no-reply@spouseprogram.fi email
Goto career track page and sing up (in the bottom of page), fill in form and check that admin user gets email (with sane headers)
